app/pages/learn/protocols/aura.mdx

```diff
@@ -0,0 +1,354 @@
+import { Callout } from "nextra/components"
+
+# Aura
+
+https://aura.finance/
+
+**Deployments**
+
+| Name                              | Address                                                                                                               |
+| :-------------------------------- | :-------------------------------------------------------------------------------------------------------------------- |
+| `Booster`                         | [0xA57b8d98dAE62B26Ec3bcC4a365338157060B234](https://etherscan.io/address/0xA57b8d98dAE62B26Ec3bcC4a365338157060B234) |
+| `RewardPoolDepositWrapper`        | [0xB188b1CB84Fb0bA13cb9ee1292769F903A9feC59](https://etherscan.io/address/0xB188b1CB84Fb0bA13cb9ee1292769F903A9feC59) |
+| `auraBAL`                         | [0x616e8BfA43F920657B3497DBf40D6b1A02D4608d](https://etherscan.io/address/0x616e8BfA43F920657B3497DBf40D6b1A02D4608d) |
+| `BalDepositorWrapper`             | [0x68655AD9852a99C87C0934c7290BB62CFa5D4123](https://etherscan.io/address/0x68655AD9852a99C87C0934c7290BB62CFa5D4123) |
+| `AuraBalBaseRewardPool`           | [0x00A7BA8Ae7bca0B10A32Ea1f8e2a1Da980c6CAd2](https://etherscan.io/address/0x00A7BA8Ae7bca0B10A32Ea1f8e2a1Da980c6CAd2) |
+| `AuraBalStaker`                   | [0xa3fCaFCa8150636C3B736A16Cd73d49cC8A7E10E](https://etherscan.io/address/0xa3fCaFCa8150636C3B736A16Cd73d49cC8A7E10E) |
+| `stkauraBAL`                      | [0xfAA2eD111B4F580fCb85C48E6DC6782Dc5FCD7a6](https://etherscan.io/address/0xfAA2eD111B4F580fCb85C48E6DC6782Dc5FCD7a6) |
+| `AuraBalBptDepositor`             | [0xeAd792B55340Aa20181A80d6a16db6A0ECd1b827](https://etherscan.io/address/0xeAd792B55340Aa20181A80d6a16db6A0ECd1b827) |
+| `AuraBalVirtualBalanceRewardPool` | [0xAc16927429c5c7Af63dD75BC9d8a58c63FfD0147](https://etherscan.io/address/0xAc16927429c5c7Af63dD75BC9d8a58c63FfD0147) |
+| `vlAURA`                          | [0x3Fa73f1E5d8A792C80F426fc8F84FBF7Ce9bBCAC](https://etherscan.io/address/0x3Fa73f1E5d8A792C80F426fc8F84FBF7Ce9bBCAC) |
+| `AURA`                            | [0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF](https://etherscan.io/address/0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF) |
+
+## Action: `deposit`
+
+**Parameters**
+
+| Name      | Type                  | Description                            |
+| :-------- | :-------------------- | :------------------------------------- |
+| `targets` | _string[] (required)_ | `Pool` name \| `Pool` bpt \| `Pool` id |
+| `tokens`  | _string[] (optional)_ | `Token` symbol \| `Token` address      |
+
+### Allowed functions
```

Comment:
since it's the only sub section I'd actually remove this headline (and pull all function section one level up)